load_dumps.php

```diff
@@ -0,0 +1,25 @@
+<?php
+require('app/config.php');
```

Comment:
Also i think you can avoid this line as the framework automatically includes the config.php file in all the php scripts. But check it once anyways